client/src/index.css

```diff
@@ -55,3 +55,7 @@ h6 {
 .chat blockquote {
 	@apply px-2 my-4 border-l-4 border-gray-600;
 }
+
+.message-reply-container * {
+	@apply m-0 p-0;
+}
```

Comment:
no more classes except absolutely necessary like when you can't access the element, like styling a library element you can't see in code but can see in inspect

Reply:
In this case, it is absolutely necessary, to remove unexpected default margins applied by MarkdownIt.